.github/workflows/operator-integration-tests-v2.yml

```diff
@@ -202,7 +202,92 @@ jobs:
         working-directory: ./api
     outputs:
       matrix: ${{ steps.set-matrix.outputs.matrix }}
-  
+
+  now-prep-testbed:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout now repo
+        uses: actions/checkout@v3
+        with:
+          repository: jina-ai/now
+          fetch-depth: 0
+      - name: Get now latest tag
+        id: latest
+        uses: "WyriHaximus/github-action-get-previous-tag@v1"
+    outputs:
+      tag: ${{ steps.latest.outputs.tag }}
+
+  now-integration-tests:
+    needs: now-prep-testbed
+    runs-on: ubuntu-latest
+    steps:
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: us-east-1
+      - name: Read secrets from AWS Secrets Manager into environment variables
+        uses: abhilash1in/aws-secrets-manager-action@v2.1.0
+        with:
+          secrets: |
+            /wolf/nprd/general
+          parse-json: true
+      - name: Checkout now repo
+        uses: actions/checkout@v3
+        with:
+          repository: jina-ai/now
+          ref: ${{ needs.now-prep-testbed.outputs.tag }}
+      - name: Setup Python 3.7
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.7
+      - name: Prepare environment
+        run: |
+          sudo apt update && sudo apt install ffmpeg
+          python -m pip install --upgrade pip
+          python -m pip install wheel
+          export JCLOUD_LOGLEVEL=DEBUG
+          pip install --no-cache-dir ".[full,test]"
+      - name: Add JCloud auth token
+        run: |
+          mkdir -p ~/.jina
+          touch ~/.jina/config.json
+          echo "{\"auth_token\": \"${_WOLF_NPRD_GENERAL_WOLF_TOKEN}\"}" > ~/.jina/config.json
+          cat ~/.jina/config.json
```

Comment:
This is an old way of setting tokens programmatically. Please use `JINA_AUTH_TOKEN` env var.

https://github.com/jina-ai/jina-hubble-sdk/blob/ea43ea0c0a04ea2952a96de464e7fa6632fef504/hubble/utils/auth.py#L168

Reply:
`now` looks for that file which causes the worfklow to fail. https://github.com/jina-ai/now/blob/main/now/utils.py#L127
https://github.com/jina-ai/cloud-ops/actions/runs/3675274819/jobs/6214508638#step:7:135